x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/factories/utils/strip_non_ecs_fields.test.ts

```diff
@@ -551,72 +551,4 @@ describe('stripNonEcsFields', () => {
       expect(removed).toEqual([]);
     });
   });
-
-  // geo_point is too complex so we going to skip its validation
-  describe('geo_point field', () => {
```

Comment:
can we keep this file? It's not needed to be reverted

Reply:
done!